.travis.yml

```diff
@@ -1,12 +1,13 @@
-language: node_js
-node_js:
-- '0.10'
+language: dart
+dart:
+- stable
+with_content_shell: true
 before_install:
 - npm install -g bower
 - npm install -g grunt-cli
-install:
-- ./init.sh
-script:
+- npm install
+- bower install
+before_script:
```

Comment:
...so, instead of recreating those tasks in our travis.yml, I just changed when we run grunt, and stop using init.sh.  Init.sh's git cleaning blew away all the dart sdk stuff that is done in the language setup.